src/gui/qgsgui.h

```diff
@@ -47,6 +49,13 @@ class GUI_EXPORT QgsGui
     static QgsGui *instance();
 
     /**
+     * Returns the global native interface, which offers abstraction to the host OS's underlying public
+     * interfaces.
+     * \note Not available in Python bindings
+     */
+    SIP_SKIP static QgsNative *nativePlatformInterface();
```

Comment:
I wasn't sure, so I decided to save the work until there was a demonstrated need for it